src/components/playback/playbackmanager.js

```diff
@@ -3680,6 +3693,10 @@ export class PlaybackManager {
         return this.getCurrentTicks(player) / 10000;
     }
 
+    getNextItem() {
+        return this._playQueueManager.getNextItemInfo();
```

Comment:
Do we need to do anything for player managed queues? Or does the `playQueueManager` handle that logic?